packages/react-aria-components/src/GridList.tsx

```diff
@@ -228,9 +228,7 @@ export interface GridListItemProps<T = object> extends RenderProps<GridListItemR
   /** The object value that this item represents. When using dynamic collections, this is set automatically. */
   value?: T,
   /** A string representation of the item's contents, used for features like typeahead. */
-  textValue?: string,
-  /** An accessibility label for this item. */
-  'aria-label'?: string
```

Comment:
Removed because the grid item hooks doesn't use the provided `aria-label` anyways, instead opting to use the `textValue`. I don't quite remember why this differs with the menu and listbox item hooks, feels inconsistent

Reply:
Won't this still be an API change?

Reply:
indeed it will be, but seeing as providing an `aria-label` doesn't do anything I figured we could remove it as a bug